### ISSUE TITLE
No such file or directory 'vim' error

### ISSUE THREAD

**Issue by ghost** [WnP/vimwiki_markdown#3]
My init.vim:
```
          let g:vimwiki_list = [{
	\ 'path': '/media/vimwiki',
	\ 'template_path': '/media/vimwiki/templates/',
	\ 'template_default': 'default',
	\ 'syntax': 'markdown',
	\ 'ext': '.md',
	\ 'path_html': '~/vimwiki/site_html/',
	\ 'custom_wiki2html': 'vimwiki_markdown',
	\ 'template_ext': '.tpl'}]
```
Here's the error:
```
Vimwiki: Saving Vimwiki files ...
Vimwiki: Deleting non-wiki html files ...
Vimwiki: Converting wiki to html files ...
Vimwiki: Processing /media/vimwiki/Markdowncheatsheet.md
Traceback (most recent call last):^@  File "/usr/bin/vimwiki_markdown", line 5, in <module>^@    from vimwiki_markdo
wn import main^@  File "/usr/lib/python3.8/site-packages/vimwiki_markdown.py", line 32, in <module>^@    with subpro
cess.Popen(^@  File "/usr/lib/python3.8/subprocess.py", line 854, in __init__^@    self._execute_child(args, executa
ble, preexec_fn, close_fds,^@  File "/usr/lib/python3.8/subprocess.py", line 1702, in _execute_child^@    raise chil
d_exception_type(errno_num, err_msg, err_filename)^@FileNotFoundError: [Errno 2] No such file or directory: 'vim'^@
Vimwiki: Processing /media/vimwiki/Vim.md
Traceback (most recent call last):^@  File "/usr/bin/vimwiki_markdown", line 5, in <module>^@    from vimwiki_markdo
wn import main^@  File "/usr/lib/python3.8/site-packages/vimwiki_markdown.py", line 32, in <module>^@    with subpro
cess.Popen(^@  File "/usr/lib/python3.8/subprocess.py", line 854, in __init__^@    self._execute_child(args, executa
ble, preexec_fn, close_fds,^@  File "/usr/lib/python3.8/subprocess.py", line 1702, in _execute_child^@    raise chil
d_exception_type(errno_num, err_msg, err_filename)^@FileNotFoundError: [Errno 2] No such file or directory: 'vim'^@
Vimwiki: Processing /media/vimwiki/index.md
Traceback (most recent call last):^@  File "/usr/bin/vimwiki_markdown", line 5, in <module>^@    from vimwiki_markdo
wn import main^@  File "/usr/lib/python3.8/site-packages/vimwiki_markdown.py", line 32, in <module>^@    with subpro
cess.Popen(^@  File "/usr/lib/python3.8/subprocess.py", line 854, in __init__^@    self._execute_child(args, executa
ble, preexec_fn, close_fds,^@  File "/usr/lib/python3.8/subprocess.py", line 1702, in _execute_child^@    raise chil
d_exception_type(errno_num, err_msg, err_filename)^@FileNotFoundError: [Errno 2] No such file or directory: 'vim'^@
Vimwiki: HTML exported to /media/vimwiki/site_html/
Vimwiki: Done!

```
I use neovim - maybe that's the problem?

**Comment by WnP** [comment]
Hi @MidlifeHeart 

I also use neovim, by the way I've aliased `vim` to spawn neovim on my setup, that's why I didn't hit that issue by myself.

As a quick fix you can do the same (just add `alias vim=nvim` to your bashrc or zshrc).

I'll do a clean fix as soon as possible and keep you in touch.

Thanks for reporting this issue.

**Comment by WnP** [comment]
Hi @MidlifeHeart,

This issue has been fixed in [vimwiki-markdown 0.3.1](https://pypi.org/project/vimwiki-markdown/#history).
